lib/emulationHandler.js

```diff
@@ -0,0 +1,19 @@
+const setEmulation = async (emulation) => {
+    this._emulation = emulation;
+};
+
+const setGeolocation = async(options) => {
```

Comment:
Can we call this `setLocation` because the browser popup uses the term location.
For e.g
<img width="326" alt="screenshot 2019-01-23 at 10 09 19 am" src="https://user-images.githubusercontent.com/54427/51583547-20597600-1ef7-11e9-985a-a083652edc91.png">

Reply:
Done